recOrder/compute/qlipp_compute.py

```diff
@@ -258,7 +258,7 @@ def reconstruct_phase2D(S0, recon, method='Tikhonov', reg_p=1e-4, rho=1,
 
     """
 
-    _, phase2D = recon.Phase_recon(np.copy(S0), method=method, reg_p=reg_p, rho=rho, lambda_p=lambda_p, itr=itr, verbose=False)
+    _, phase2D = recon.Phase_recon(np.copy(S0).astype('float'), method=method, reg_p=reg_p, rho=rho, lambda_p=lambda_p, itr=itr, verbose=False)
 
```

Comment:
When the input dataset has a non-float type (uint), it will throw an error when normalizing the intensity in the `inten_normalization` function. I have added this conversion in the ` reconstruct_phase2D` and ‘reconstruct_phase3D’ functions in ‘qlipp_compute.py’

Reply:
Thanks for catching this!